portfolio/src/main/webapp/script.js

```diff
@@ -66,7 +66,8 @@ function tryHarderAndGiveDaisy() {
 
 async function getHelloFromButton() {
     const response = await fetch('/data');
-    const text = await response.text();
+    const text = await response.json();
+    const messageToDisplay = text[Math.floor(Math.random() * (text.length))];
```

Comment:
Can you instead make this code show all returned messages? You could create a new `<p>` element for each message and inject the fetched text.